src/com/facebook/buck/cli/TestCommand.java

```diff
@@ -120,6 +120,9 @@
   @Nullable
   private Boolean isBuildFiltered = null;
 
+  @Option(name = "--build-only", usage = "Only build test targets without running tests.")
+  private boolean shouldBuildOnly = false;
```

Comment:
nit: `should` -> `is` to match with rest of flags, and `boolean -> @Nullable Boolean` maybe?